packages/documentation-site/patternfly-docs/content/get-started/community/community.md

```diff
@@ -70,26 +70,26 @@ import CalendarIcon from '@patternfly/react-icons/dist/esm/icons/calendar-alt-ic
     </Card>
   </Grid>
 
-  <Title size="3xl" className="pf-v5-u-mb-sm ws-page-title pf-v5-u-mt-3xl" headingLevel="h2">Creating new communities</Title>
+  <Title size="3xl" className="pf-v6-u-mb-sm ws-page-title pf-v6-u-mt-3xl" headingLevel="h2">Creating new communities</Title>
```

Comment:
I don't know how much it matters, but it's worth noting there are a lot of utility classes used on this page where it looks like we have a stylesheet for other pages? I wonder if we should be more consistent. Also I wonder if we used `<Text>` headings, if we would need all of these margins and hwhat-not on the headings.

Reply:
Followup https://github.com/patternfly/patternfly-org/issues/4021